pkg/api/v1/schema.go

```diff
@@ -114,6 +114,10 @@ type ResourceRequirements struct {
 	// Valid resource keys are "memory" and "cpu".
 	// +optional
 	Limits v1.ResourceList `json:"limits,omitempty"`
+	// Don't ask the scheduler to take the guest-management overhead into account. Instead
+	// put the overhead only into the requested memory limits. This can lead to crashes if
+	// all memory is in use on a node. Defaults to false.
+	OvercommitGuestOverhead bool `json:"overcommitGuestOverhead,omitempty"`
```

Comment:
What use case does it have?
can you also add `// +optional`

Reply:
It allows a higher vm density. If you expose 32gb of ram for the k8s scheduler, you can schedule 8x4gb vms to the node. Under the assumptions that not all vms use all their memory at the same time you vms should mostly work normal then.

Reply:
Thanks for the explanation I forgot that we have `-realtime mlock=off`.